blueprint/index.md

```diff
@@ -7,232 +7,285 @@ icon: blueprint
+7. Name the resource as `postutterance` and click **Create Resource**.
+   ![Create Resource](images/create-resource.png "Create Resource")
+8. Choose **Actions** > **Create Method**. Select **POST** from the list and click the check mark to confirm.
+   ![Add POST Method to Gateway](images/MethodSettings.png "Add POST Method to Gateway")
+9.  Select **Lambda Function** as the Integration type, enter the function name, and click **Save**.
 	![Trigger Lambda Settings](images/LambdaSettings.png "Trigger Lambda Settings")
-	8. Scroll down to Mapping Templates. Click Add mapping template. Enter application/json. Click the tick and scroll down to see a new box
-	
+
+10. Click **Ok**.
+
+11. Select **Integration Request** and expand **Mapping Templates**.
+        
+12. Click **Add mapping template** and enter `application/json` as the MIME-type for the **Content-Type**. Click the create icon next to the box.
+    
 	![Set Mapping Template](images/MappingTemplate.png "Set Mapping Template")
-	9. Paste this code into the code box as shown above
-	
-{
-    "body": $input.json('$'),
-    "headers": {
-        #foreach($param in $input.params().header.keySet())
-        "$param": "$util.escapeJavaScript($input.params().header.get($param))"
-        #if($foreach.hasNext),#end
-        #end
-    }
-}
-
-	10. Request Body Passthrough should be set to "Never"
-	11. Click Save.
-	12. Click the Actions button again and select Deploy API
 
+13. Select **Never** for **Request body passthrough** option.
+    
+14. Scroll down and enter the following JSON request into the code box and click **Save**:
+	```JSON	
+	{
+		"body": $input.json('$'),
+		"headers": {
+			#foreach($param in $input.params().header.keySet())
+			"$param": "$util.escapeJavaScript($input.params().header.get($param))"
+			#if($foreach.hasNext),#end
+			#end
+		}
+	}
+	```
+15. For the POST method, choose **Actions** > **Deploy API**.
+    
 	![Deploy API Gateway](images/DeployAPI.png "Deploy API Gateway")
-	13. You can now close the tab and go back to the Lambda function.
-	14. Under Configuration, open the Triggers and the API Gateway. Click on Details and record the API Endpoint url for later
-	
+
+16. Go back to the Lambda function. Under the **Configuration** tab, select **Triggers**. Expand the **Details** section and make a note of the API endpoint URL for later use.
+    
 	![Trigger URL](images/TriggerURL.png "Trigger URL")
 
-###Configure Genesys Cloud
 
-1.Create a new “Genesys Bot Connector” integration
-	1. Go to Admin and select Integrations.
-	2. Click on the +Integrations button to add a new one. Search for Genesys Bot Connector and click Install
-	3. Name the integration and click on Configuration
-	4. In the field Bot Connector Handle Utterance URI paste in the API endpoint url saved from the AWS API Gateway config e.g. https://accountname.execute.api.eu.west-1.amazonaws.com/default/postutterance
+### Configure Genesys Cloud
```

Comment:
Please verify the heading styles. It looks like this H3 has no text under it. Is it needed?